date: '10-06-2021'
sections:
  security_fixes:
    - Los paquetes se han actualizado a las últimas versiones de seguridad.
  bugs:
    - "El SVN 1.7 y los clientes más antiguos mostraron un error cuando se utilizaron los comandos \"svn co\" y \"svn export\"."
    - "El acceso a un repositorio mediante el shell administrativo utilizando \"ghe-repo <owner>/<reponame>\" provocaba bloqueos."
    - "Después de la actualización, los usuarios experimentaron disponibilidad reducida durante el uso intensivo, porque los servicios se reiniciaron con demasiada frecuencia. Esto sucedería a causa del error de coincidencia de tiempo de expiración entre la configuración nómada y la de los servicios internos."
    - "En algunas instancias, la ejecución de \"ghe-repl-status\" después de haber configurado las Acciones de GitHub produjo un error y \"ghe-actions-teardown\" falló."
    - '"ghe-dbconsole" devolvió errores en algunas circunstancias.'
    - "Los errores de importación de organizaciones o repositorios desde orígenes distintos a GitHub podrían producir un error de tipo \"método sin definir \"[]\" para nil:NilClass\"."
    - "Los nombres de perfil de GitHub podrían haber cambiado accidentalmente al utilizar la autenticación de SAML, si el nombre de perfil de GitHub no coincidió con el valor del atributo que se asignó en el campo \"Full name\" de la consola de administración."
    - "La actualización de una instancia que previamente ejecutó la versión 2.13, pero no la versión 2.14, daba como resultado un error de migración de base de datos que se relacionaba con la transición de datos \"AddRepositoryIdToCheckRuns\"."
  changes:
    - "Los usuarios de GraphQL API pueden consultar el campo público \"closingIssuesReferences\" en el objeto \"PullRequest\". Este campo recupera incidencias que se cerrarán de forma automática cuando se combine la solicitud de incorporación de cambios relacionada. Este enfoque también permitirá migrar estos datos en el futuro, como parte de un proceso de migración más fiel."
  known_issues:
    - "El registro npm de {% data variables.product.prodname_registry %} ya no devuelve un valor de hora en las respuestas de metadatos. Esto se hacía para permitir mejoras de rendimiento importantes. Seguimos teniendo todos los datos necesarios para devolver un valor de tiempo como parte de la respuesta de metadatos y reanudaremos la devolución de este valor en el futuro una vez que hayamos resuelto las incidencias de rendimiento existentes."
    - "En una instalación nueva de {% data variables.product.prodname_ghe_server %} que no tenga ningún usuario, cualquier atacante podría crear el primer usuario administrativo."
    - Las reglas de cortafuegos personalizadas se eliminan durante el proceso de actualización.
    - "Los archivos supervisados de Git LFS [cargados desde la interface web](https://github.com/blog/2105-upload-files-to-your-repositories) se han agregado de manera incorrecta y directa al repositorio."
    - Las incidencias no se pueden cerrar si contienen un enlace permanente a un blob del mismo repositorio en el que la ruta es mayor de 255 caracteres.
    - "Cuando se habilita \"Los usuarios pueden buscar en GitHub.com\" con GitHub Connect, las incidencias en los repositorios privados e internos no se incluyen en los resultados de la búsqueda de GitHub.com."
    - |Después de actualizar de la versión 3.0.x a la 3.1.x, en algunos casos, puede que Acciones de GitHub falle con un error de tipo: "An unexpected error occurred when executing this workflow". Para solucionar este problema, conéctate al shell administrativo (ssh) y ejecuta:

```

ghe-actions-console -s actions -c "Queue-ServiceJob -JobId 4DB1F4CF-19FD-40E0-A253-91288813DE8B"

```

|
    - "Cuando un nodo de réplica está sin conexión en una configuración de disponibilidad alta, es posible que {% data variables.product.product_name %} todavía enrute las solicitudes a {% data variables.product.prodname_pages %} al nodo sin conexión, lo que reduce la disponibilidad de {% data variables.product.prodname_pages %} para los usuarios."
    - "Si se habilitan las {% data variables.product.prodname_actions %} para {% data variables.product.prodname_ghe_server %}, desmontar un nodo de réplica con \"ghe-repl-teardown\" tendrá éxito, pero podría devolver un \"ERROR:Running migrations\"."
    - Los límites de recursos que son específicos para procesar enlaces previos a la recepción podrían ocasionar errores en algunos de ellos.
